README.rst

```diff
@@ -7,6 +7,17 @@ XBlock Flow Control
 
 The Flow Control XBlock provides a way to display the content of a unit or to redirect the user elsewhere based on compliance with a condition that evaluates the submission or the score of a problem or a set of problems.
 
+Compatibility Notes
+===================
+
++------------------+--------------+
+| Open edX Release | Version      |
++==================+==============+
+| Redwood          | >= 2.0.0     |
++------------------+--------------+
+| Sumac            | >= 2.0.0     |
++------------------+--------------+
+
 Installing on Open edX Devstack
```

Comment:
I think we should remove this section, it's no longer relevant
https://github.com/eduNEXT/flow-control-xblock/blob/3cce94718243504fb36a8fbeec5eeebb3563293a/README.rst#installing-on-open-edx-devstack